backend/src/services/mail-verification/send-activation-link/send-link.ts

```diff
@@ -16,23 +14,12 @@ const sendLink = async (email: string, token: string): Promise<void> => {
     },
   });
 
-  const handlebarOptions: hbs.NodemailerExpressHandlebarsOptions = {
-    viewEngine: {
-      extname: '.hbs',
-      defaultLayout: false,
-      partialsDir: templatePath,
-    },
-    viewPath: templatePath,
-    extName: '.hbs',
-  };
-
-  transporter.use('compile', hbs(handlebarOptions));
-
   const options = {
     from: ENV.MAIL.FROM_EMAIL_VALIDATE,
     to: email,
     subject: MailActivate.SUBJECT,
     template: 'email',
+    html: templateService.getMessage(`${MailActivate.ACTIVATE_URL}${token}`),
     context: {
       link: `${MailActivate.ACTIVATE_URL}${token}`,
     },
```

Comment:
Do we still need the `context`? I see that we already pass the `link` as the param into the `getMessage`

Reply:
you are right, now we don't need context. 
deleted.